### PR TITLE
GET & POST `/admin/expense_rules` and POST `/admin/expense_rules/delete`

### PR DESCRIPTION
**Pull request by muleyashutosh**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=9ayNxrl)

### CODE REVIEW

File: src/admin/paths/admin@expense_rules.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              count:
+                description: |
+                  The total number of objects that can be requested.
```
Comment (sanskarsharma) [inline src/admin/paths/admin@expense_rules.yaml:69]: use src/components/schemas/count.yaml

File: src/admin/paths/admin@expense_rules.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                type: integer
+                example: 10000
+              offset:
```
Comment (sanskarsharma) [inline src/admin/paths/admin@expense_rules.yaml:72]: use src/components/schemas/offset.yaml

**Review by sanskarsharma — APPROVED** [review]

File: src/admin/paths/admin@expense_rules.yaml (hunk 2 above)
Comment (muleyashutosh) [reply src/admin/paths/admin@expense_rules.yaml:69]: done

File: src/admin/paths/admin@expense_rules.yaml (hunk 1 above)
Comment (muleyashutosh) [reply src/admin/paths/admin@expense_rules.yaml]: done

**Pull request merged**